user-guide/with-ingress-gce.md

```diff
@@ -0,0 +1,163 @@
+
+Read up on [managed certificates on GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs). You need
+a DNS name and point it to the external IP of the load balancer. Going forward it is assumed that this DNS name
+is www.example.com .
+
+certificate.yaml:
+```yaml 
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: www-example-com
+spec:
+  domains:
+    - www.example.com
+```
+
+Modify the ingress from before:
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: basic-ingress
+  annotations:
+    networking.gke.io/managed-certificates: www-example-com
+spec:
+  backend:
+    serviceName: ambassador
+    servicePort: 8080
+```
+
+Please wait (5-15 minutes) until the certificate is created and all edge servers have the certificates ready. 
+`kubectl describe ManagedCertificate` will show you the status or go to the web console to view the load balancer.
+
+You should now be able to access the web service via https://www.example.com
+
+## 5. Configure Ambassador to do HTTP -> HTTPS redirection
+
+### Redirecting the health check
+The google load balancer depends on a successfull health check to route traffic to your nodes. By default the
+health check will send a HTTP request to `/` and expect `HTTP 200` to be returned. Otherwise the node is not healthy and if there
+are no healthy nodes the load balancer will return a 500 error.
+
+The health check definition needs to be changed. Ambassador will respond with a HTTP redirect after enabling HTTP->HTTPS redirection.
+The health check would fail. This is where the `ambassador-admin` service is coming into play. It can be queried at `/ambassador/v0/check_ready`
+and will return the correct HTTP 200 answer.
+
+- determine the node port from the service ambassador-admin: `kubectl get service`. The NodePort is a port number between 30000 and 32767.
+- Open the cloud console http://console.cloud.google.com and select Network Services/Load Balancing.
+- Click on the load balancer
+- In the Backend Section you will find a link to the health check
+- Click it and edit the health check
+  - Change the port to the NodePort from ambassador-admin
+  - Change the Request path to `/ambassador/v0/check_ready`
+  - Optional: change the check interval to their defaults (10 seconds check interval, 5 s timeout, unhealthy 3 tries)
+
+Now the service health is determined by contacting ambassador-admin service
+
+### Enabling HTTP -> HTTPS 
+
+- Configure Ambassador to [redirect traffic from HTTP to HTTPS](../reference/tls/cleartext-redirection.md#protocol-based-redirection). 
+- you need to restart ambassador to effect the changes
+
+The result should be that http://www.example.com will redirect to https://www.example.com. 
+
+You can now add more services by specifying the hostname in the mapping. 
+
+## Related Topics
+
+- [Configuring xff_num_trusted_hops](../reference/core/ambassador.md#xff_num_trusted_hops)
```

Comment:
Did you end up configuring this? I'm guessing you need to based on the fact that you mentioned it, but it's not noted anywhere else in the doc.

Reply:
I did configure it, but in the end the x-forwarded-for header just is a list of IPs so I took the first one. I deleted the reference since it is not really relevant for the google load balancer case